Farfetch/loadshedding
# feat: Report queue metrics for no waiting tasks

# Description

Reports the waiting time for the tasks with no waiting time, so that the histogram reflects all the waiting times.

Update the current queue size to all processed tasks.

Closes #16 

## How Has This Been Tested?

1. Run the Samples.WebApi
2. Call the weather endpoint once
3. Call the metrics endpoint and the response should have `http_requests_queue_items_total` and `http_requests_queue_time_seconds` with the value 0 for the `normal` priority and `GET` method.

## Checklist


### Disclaimer

By sending us your contributions, you are agreeing that your contribution is made subject to the terms of our [Contributor Ownership Statement](https://github.com/Farfetch/.github/blob/master/COS.md)
